WorkloadManagementSystem/Service/WMSUtilities.py

```diff
@@ -77,29 +70,31 @@ def getCREAMPilotOutput( proxy, pilotRef, pilotStamp ):
   result['StdErr'] = error
   return result
 
-def theARCJob(theCE, theArcID):
-  # Create an ARC Job with all the needed / possible parameters defined.
-  # By the time we come here, the environment variable X509_USER_PROXY should already be set
+def ARCJob( theCE, theArcID ):
+  """ Create an ARC Job with all the needed / possible parameters defined.
+      By the time we come here, the environment variable X509_USER_PROXY should already be set
+  """
   j = arc.Job()
   j.JobID = theArcID
-  statURL = "ldap://%s:2135/Mds-Vo-Name=local,o=grid??sub?(nordugrid-job-globalid=%s)" % (theCE, theArcID)
-  j.JobStatusURL = arc.URL(statURL)
+  statURL = "ldap://%s:2135/Mds-Vo-Name=local,o=grid??sub?(nordugrid-job-globalid=%s)" % ( theCE, theArcID )
```

Comment:
Isn't this supposed to be kind of a permanent port number? Note that this is not a DIRAC service, but ldap. A similar usage is for example in line 81 in Core/Utilities/Grid.py

Let me know if this should not be hardcoded, and I will be happy to put it in the configuration. In which case, I may want to make a more generic function for searching in a prioritised list through the configuration.